tests/test_circular_imports.py

```diff
@@ -1,22 +1,32 @@
-"""Tests for circular imports in all local packages and modules.
+# -*- coding: utf-8 -*-
```

Comment:
This is unnecessary in py3-only codebase

Reply:
Will remove within `README` PR now :)

Reply:
You could have a `local` hook in the `pre-commit` config.